src/url-utils.ts

```diff
@@ -19,50 +19,50 @@ import * as path from 'path';
 import * as url from 'url';
 import {parseUrl} from 'polymer-analyzer/lib/core/utils';
 import constants from './constants';
+import {FileRelativeUrl, ResolvedUrl} from 'polymer-analyzer';
 
 const sharedRelativeUrlProperties =
     ['protocol', 'slashes', 'auth', 'host', 'port', 'hostname'];
 
 /**
  * A string representing a URL.
  */
-export type UrlString = string;
-
-export function ensureTrailingSlash(href: UrlString): UrlString {
-  return href.endsWith('/') ? href : href + '/';
+export function ensureTrailingSlash(href: string) {
```

Comment:
Update comment?

Reply:
Right.  Done.  Also updated type definition so that it can return the same type of `href` that comes in:

```ts
/**
 * Given a string representing a URL or path of some form, append a `/`
 * character if it doesn't already end with one.
 */
export function ensureTrailingSlash<T>(href: T): T {
  const hrefString = href as any as string;
  return hrefString.endsWith('/') ? href : (href + '/') as any as T;
}
```